client/components/post-schedule/docs/example.jsx

```diff
@@ -169,7 +169,6 @@ const PostScheduleExample = localize(
 											position: 'relative',
 										} }
 										onClick={ this.clearState.bind( this, 'timezone' ) }
-										href="#"
```

Comment:
Could you please document why the `href="#"` attributes were removed? This feels like it might have unintended consequences, so please remove if it wasn't something you intended to include.